cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/store/AppMetadataStore.java

```diff
@@ -57,12 +57,12 @@
     GSON = builder.create();
   }
 
-  private static final String TYPE_APP_META = "appMeta";
-  private static final String TYPE_STREAM = "stream";
-  private static final String TYPE_RUN_RECORD_STARTED = "runRecordStarted";
-  private static final String TYPE_RUN_RECORD_COMPLETED = "runRecordCompleted";
+  public static final String TYPE_APP_META = "appMeta";
+  public static final String TYPE_STREAM = "stream";
+  public static final String TYPE_RUN_RECORD_STARTED = "runRecordStarted";
+  public static final String TYPE_RUN_RECORD_COMPLETED = "runRecordCompleted";
   private static final String TYPE_PROGRAM_ARGS = "programArgs";
```

Comment:
Do we need to ? The Upgrade doc does not mention anything about it and I didn't see them in the hbase table too. I will look into it. It might be a good idea to keep the upgrade doc requirements updated if we find new things as we proceed.